config/config.js

```diff
@@ -67,6 +67,15 @@ var validateEnvironmentVariable = function () {
 };
 
 /**
+ * Validate config.domain is set
+ */
+var validateDomainIsSet = function (config) {
+  if(!config.app.domain){
+    console.error(chalk.red('+ Important warning: config.domain is empty. To prevent HTTP Host header attacks please set config.domain.'));
```

Comment:
To be consistent, I don't think this should be `console.error`, just `console.log`.